src/components/spacing-control/index.js

```diff
@@ -20,6 +20,8 @@ const SpacingControl = props => {
 	// Allow different sets of min/max/step values per screen size.
 	const min = Array.isArray( props.min ) ? props.min : [ props.min, props.min, props.min ]
 	const max = Array.isArray( props.max ) ? props.max : [ props.max, props.max, props.max ]
+	const sliderMin = Array.isArray( props.sliderMin ) ? props.sliderMin : [ props.sliderMin, props.sliderMin, props.sliderMin ]
```

Comment:
I don't think you need to specify the `sliderMin` and `sliderMax` values here. From what I know the range control should be able to handle things as expected if the `sliderMin` and `sliderMax` values are not provided.

Reply:
We need to add this in order to pass the corresponding`sliderMin` and`sliderMax` value inside the `FourRangeControl` component

Reply:
Additional note: Similar to `min` and `max`, `sliderMin` and `sliderMax` also handle number values `e.g. sliderMax={ 100 }`